### ISSUE TITLE
Enable and Require 2FA for all wordpress sites

### ISSUE THREAD

**Issue by Firefishy** [openstreetmap/operations#856]
We run a few wordpress sites. To improve security we should enable 2FA.

May need to find a suitable (supported) plugin to enable 2FA in wordpress.

* blog.openstreetmap.org
* 2007.stateofthemap.org
* 2008.stateofthemap.org
* 2009.stateofthemap.org
* 2010.stateofthemap.org
* 2011.stateofthemap.org
* 2012.stateofthemap.org
* join.osmfoundation.org (civicrm)

**Comment by tomhughes** [comment]
Can we just disable logins for the stateofthemap ones maybe? No idea if that's possible but they're all just archived basically.

**Comment by Firefishy** [comment]
> Can we just disable logins for the stateofthemap ones maybe? No idea if that's possible but they're all just archived basically.

Yes, it should be possible to do it using the [wp-cli](https://developer.wordpress.org/cli/commands/user/) (remove cap).

**Comment by Firefishy** [comment]
As an early step I have emailed all blog.openstreetmap.org user asking if they still use their accounts. Quite a few bounces.

**Comment by pnorman** [comment]
Do we have a plugin identified for 2fa yet?

**Comment by Firefishy** [comment]
We are using https://wordpress.org/plugins/wp-2fa/ and it has been live on blog.openstreetmap.org for a few months.

**Comment by pnorman** [comment]
So the to-do on this is install it on all sites, configure it on all sites, set a deadline, and notify users?

**Comment by Firefishy** [comment]
> So the to-do on this is install it on all sites, configure it on all sites, set a deadline, and notify users?

Yes, but only for join.osmfoundation.org. The other legacy sites my preference would be to just disable all users.
